src/lib/util.ts

```diff
@@ -1,7 +1,9 @@
-import { KeyboardEvent } from "react";
+import { KeyboardEvent, memo } from "react";
 import { AnyJson, JsonMap, MIDIChannelPressureMetaMapping, MIDIControlChangeMetaMapping, MIDIKeypressMetaMapping, MIDIMetaMapping, MIDINoteMetaMapping, MIDIPitchBendMetaMapping, MIDIProgramChangeMetaMapping, OSCQueryStringValueRange, OSCQueryValueRange } from "./types";
 import { MIDIMetaMappingType } from "./constants";
 
+export const genericMemo: <P>(component: P) => P = memo;
+
```

Comment:
unused?

Reply:
nah, it's used here in order to allow wrapping a comp with generic props:
https://github.com/Cycling74/rnbo-runner-panel/pull/192/files#diff-a1d8d95fad145f23aabf4e0f183824c6d283093b806284f792f02458286ee80eR18